### PR TITLE
add RISE and ipytree

### LINKED ISSUE #85: Add RISE and ipytree to metapackage?
RISE and ipytree are both lightweight, jupyter-related tools that I want to use very frequently in Pangeo environments. How would folks feel about adding them to this metapackage?

cc @jhamman, @scottyhq, @TomAugspurger

### PR DESCRIPTION
**Pull request by rabernat**
closes #85 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

I did not pin versions. I don't know whether that would be the right choice.

I also wonder whether https://github.com/pangeo-bot/dispatcher/blob/main/.github/workflows/watch-pangeo-notebook-feedstock.yml needs to be updated to include RISE and ipytree, or whether we can just let them get resolved at build time. Would appreciate advice on this.

### CODE REVIEW

**Comment by rabernat** [comment]
Pinging @scottyhq, @jhamman, @TomAugspurger for a review of this. I don't understand the system well enough to know if I'm doing this right.

**Comment by TomAugspurger** [comment]
Just noticing which repository this issue / PR is in. My initial reaction is that these packages would be more appropriate for whatever `environment.yaml` (or `environment.yaml`s), you're using the `pangeo-notebook` metapackage in. The reason being this metapackage is focused on providing the "core" experience. I worry slightly about a future rise / ipytree release being incompatible with a future jupyterlab / notebook release and causing issues for the entire stack building off this.

Is the primary motivation for putting it here, rather than in say https://github.com/pangeo-data/pangeo-docker-images/blob/master/pangeo-notebook/environment.yml, just wanting to have it in many different environments?

**Comment by rabernat** [comment]
I tried to explain the justification in #85. The thinking is that these packages are jupyter-related (not python related), and all of the jupyter-related machinery seems to live here.

**Comment by TomAugspurger** [comment]
Gotcha. It's a judgement call so IMO this could go either way.

How about we just merge it here and remove if it causes any dependency issues in the future? Let's give @scottyhq a few hours to respond and go ahead if we don't hear back.

**Comment by scottyhq** [comment]
Agreed there is no clear cut decision making on these things, but the initial thinking of this repository was that it should have *strictly* required packages for pangeo jupyterhub and binderhubs. i've never used rise or ipytree... are they required together? 

If we want to ensure that RISE slideshow capability is always in every environment (pangeo-notebook, ml-notebook, etc.) it should go here. If it's just another ipywidget (ipytree?) that should probably go into a specific environment.

> I did not pin versions. I don't know whether that would be the right choice. I also wonder whether https://github.com/pangeo-bot/dispatcher/blob/main/.github/workflows/watch-pangeo-notebook-feedstock.yml needs to be updated

For consistency I'd recommend pinning, and updating the bot to bump versions when released.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
```
Comment (scottyhq) [inline recipe/meta.yaml:8]: unfortunately i don't think the bot knows how to change build numbers, so if you do want to add these, it's probably easiest to put this back to 0 and use a new date for the version (2021.09.09)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,6 +17,8 @@ requirements:
     - jupyterhub-singleuser =1.4.2
     - jupyterlab =3.1.10
     - nbgitpuller =1.0.2
+    - rise
```
Comment (scottyhq) [inline recipe/meta.yaml:20]: i would suggest an explicit pin here for consistency with the other packages

**Comment by scottyhq** [comment]
closed in favor of https://github.com/pangeo-data/pangeo-docker-images/pull/256

**Pull request closed without merging**